### PR TITLE
{bio}[GCCcore/12.2.0] FLASH v2.2.00

### PR DESCRIPTION
**Pull request by eylenth**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FLASH/FLASH-2.2.00-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+homepage = 'https://ccb.jhu.edu/software/FLASH/'
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast
+and accurate software tool to merge paired-end reads from next-generation
+sequencing experiments. FLASH is designed to merge pairs of reads when the
+original DNA fragments are shorter than twice the length of reads. The
+resulting longer reads can significantly improve genome assemblies. They can
+also improve transcriptome assembly when FLASH is used to merge RNA-seq data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/f/FLASH/FLASH-2.2.00-foss-2022b.eb:15]: The Makefile looks rather straightforward, I don't see anything that suggests we need the full `foss` toolchain here. We should be able to drop this to `GCCcore`
```suggestion
toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
```

File: easybuild/easyconfigs/f/FLASH/FLASH-2.2.00-foss-2022b.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+source_urls = ['https://github.com/dstreett/FLASH2/archive/']
+sources = ['%(version)s.zip']
+checksums = ['1e54b2dd7d21ca3e0595a3ffdd27ef3098f88c4de5b9302ec5ea074b49b79960']
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/f/FLASH/FLASH-2.2.00-foss-2022b.eb:20]: (explicit `zlib` and `binutils` dependencies needed when dropping to `GCCcore` )
```suggestion

builddependencies = [('binutils', '2.39')]

dependencies = [('zlib', '1.2.12')]

```

**Review by jfgrimm — CHANGES_REQUESTED** [review]
I'd suggest dropping this to `GCCcore` since it doesn't rely on MPI/BLAS/LAPACK/FFTW.

please note that you should change the filename of the easyconfig as well to reflect that

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @eylenth!

**Pull request merged**